test/WireMock.Net.Tests/FluentMockServerTests.Proxy.cs

```diff
@@ -190,29 +193,30 @@ public partial class FluentMockServerTests
         [Fact]
         public async Task FluentMockServer_Proxy_Should_change_absolute_location_header_in_proxied_response()
         {
-            // given
-            _serverForProxyForwarding = FluentMockServer.Start();
+            // Assign
+            var settings = new FluentMockServerSettings { AllowPartialMapping = false };
+            _serverForProxyForwarding = FluentMockServer.Start(settings);
```

Comment:
Perhaps names these 2 servers more specifically as it is quite hard to follow. _serverThatReturnsRedirects and serverThatProxies ?

Reply:
I understand your comment, however for me it's clear for now. But I will think about this to see if I want to rename the these variables.